app/helpers/job_invocation_output_helper.rb

```diff
@@ -0,0 +1,33 @@
+module JobInvocationOutputHelper
+
+  def colorize_line(line)
```

Comment:
This method may look a little odd - I explain. Ansible output is colored some times and these colors are printed as regular bash color codes in live output without this method.

I haven't tried, but I'm almost certain this is also relevant to SSH. I can remove this from this PR and submit it separately.